bin/push-docker

```diff
@@ -11,25 +11,6 @@ echo $REPO
 echo $TAG
 echo '--'
 
-# Retry up to 5 times total if failed Docker login
-run=1
-count=1
-while [ $run -eq 1 ] ; do
-    echo "Trying Docker login..."
-    docker login -u $DOCKER_USER -p $DOCKER_PASS
-    if [ $? -eq 0 ]; then  # Success
-        run=0
-        break
-    fi
-    if [ $count -ge 5 ]; then
-        run=0
-        echo "Error: Exceeded retries with Docker."
-        exit 1
-    fi
-    count=$((count+1))
-    sleep 15
-done
-
 # Retry up to 5 times total if failed Docker push
 run=1
 count=1
```

Comment:
While you're at it, could change these comments below to say something like "Trying ECR push"/"Exceeded retries with ECR"

Reply:
Good catch!